### ISSUE TITLE
Stable Spawners causing lag client side

### ISSUE THREAD

**Issue by Ipiano** [brandon3055/Draconic-Evolution#238]
A couple days ago I set up a mob grinder using some stabilized spawners with wyvvern cores, nether stars, and gold apples. Everything was going great, for a while but some of my friends started complaining about lag in my area. I discounted it because I'm using a tower computer and they're using laptops; but sure enough, when I jumped on the server this morning, everything was running <10 FPS at my base, instead of the usual 25-30. I set up the spawners with a redstone signal to prevent them running, but that didn't help at all. Eventually, I just took them down (I've got enough wither heads for a while) and as soon as I logged off and logged back in, everything was back to normal.

So, to sum it up, there seems to be an issue with the stabilized spawners that causes lag issues client side; but not server side; this was only an issue for players who were in my base, not the entire server.

**Comment by Ipiano** [comment]
Oh, and in case it's relevant, I was spawning zombies, and there were two spawners in close proximity

**Comment by brandon3055** [comment]
Are you able to recreate this in a test world? Because its hard to debug something like this if it cant be recreated.

**Comment by Ipiano** [comment]
I haven't attempted it yet; I'll mess around and see if I can do anything to reproduce it

**Comment by Ipiano** [comment]
Turns out, it's not too hard to replicate. Just set up a stabilized spawner and give it redstone so it stops. Then wait. It's probably not related, but I'm also using rednet cables to move the signal. Throw a few more spawners (also stopped with a signal) in and it gets worse.

**Comment by brandon3055** [comment]
What mod pack are you using? I have not been able to replicate the issue in a test world and as far as i know noone else is having this issue.

**Comment by Ipiano** [comment]
Interesting.... It's a custom one I put together. I'll mess around a bit more and see if I can figure out what other mod is causing the issue. Like I said, I'm using rednet cable, perhaps it's just the version of MFR that I'm using

**Comment by Ipiano** [comment]
More information for you. The attached picture is what I'm doing to test this. All of the spawners have wyvvern cores, notch apples, and nether stars. The lag really seems to start when I add the nether star, but it's noticable before. I was using the following mods when I did this:

BrandonsCore 1.0.0.10-bugfix
CodeChickenCore 1.0.7.47
CoFHCore 3.1.1-324
DC 1.0.2 - Snapshot 9
MFR 2.8.1-174
neiaddons 1.12.8.29
NEIIntegration 1.1.0
NEI 1.0.5.118

Nei was only there so I could actually obtain and put down the spawners.
Additionally, the issue only seems to happen when I use the rednet cable... perhaps it's an issue on MFR side; but I haven't had any other issues with that before.

![2016-02-16_08 32 12](https://cloud.githubusercontent.com/assets/5333611/13081040/795a01b4-d488-11e5-9d5b-0c091ae4a7bb.png)

**Comment by gothroach** [comment]
I am having the exact same issue as described.  Upgraded Draconic spawners are causing my FPS to tank to <10 FPS when hooked into RedNet cables.  When hooked into other Redstone sources, this doesn't appear to happen.  It's not instant either, I can walk around for 10-15 seconds before my FPS tanks.  I'll be submitting a report to MFR also in case it's an issue with their mod.

**Comment by brandon3055** [comment]
Just letting you know this will finally be fixed in the next version. Sorry it took so long.

**Comment by Ipiano** [comment]
Understandable man. I'm currently going to school to major in Computer Science; I get how this kind of thing goes. Just glad that you know about it!
